sdk/nodejs/README.md

```diff
@@ -12,6 +12,69 @@ The Dagger NodeJS SDK contains everything you need to develop CI/CD pipelines in
 npm install @dagger.io/dagger --save-dev
 ```
 
+## Local development
+
+You may want to work on the NodeSDK and test it directly on a local node project. 
+
+1. Create a new Node project
+
+:warning: Skip this step if you already have a node project
+
+```shell
+mkdir my-test-ts-project
+
+# Init project (you may use yarn or pnpm)
+npm init -y
+
+# Add typescript
+npm install typescript --save-dev
+
+# Init typescript project
+npx tsc --init
+```
+
+2. Update project settings
+
+Dagger exports its SDK using type module so you will need to also update
+your `package.json` to the same type.
+
+Add or update the field `type` in your `package.json`
+
+```json
+"type": "module"
+```
+
+You must also update your `tsconfig.json` to use `NodeNext` as `module`.
+
+```json
+"module": "NodeNext"
+```
+
+3. Import Dagger local module
+
+Go to the Dagger nodejs SDK directory and do the following :
+
+```shell
+cd path/to/dagger/sdk/nodejs
```

Comment:
```suggestion
cd path/to/dagger/sdk/nodejs
yarn link
```

Reply:
My previous suggestion has deleted the `yarn link` my bad 🙏

Reply:
Fixed while rebasing his branch. @TomChv I forced pushed on top of your branch, to make this PR advance 😇